osu.Game/Rulesets/Mods/ModAccuracyChallenge.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using osu.Framework.Bindables;
+using osu.Framework.Localisation;
+using osu.Game.Configuration;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Overlays.Settings;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Scoring;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public class ModAccuracyChallenge : ModFailCondition, IApplicableToScoreProcessor
+    {
+        public override string Name => "Accuracy Challenge";
+
+        public override string Acronym => "AC";
+
+        public override LocalisableString Description => "Fail if your accuracy drops too low!";
+
+        public override ModType Type => ModType.DifficultyIncrease;
+
+        public override double ScoreMultiplier => 1.0;
+
+        public override Type[] IncompatibleMods => base.IncompatibleMods.Concat(new[] { typeof(ModEasyWithExtraLives), typeof(ModPerfect) }).ToArray();
+
+        public override bool RequiresConfiguration => false;
+
+        public override string SettingDescription => base.SettingDescription.Replace(MinimumAccuracy.ToString(), MinimumAccuracy.Value.ToString("##%", NumberFormatInfo.InvariantInfo));
+
+        [SettingSource("Minimum accuracy", "Trigger a failure if your accuracy goes below this value.", SettingControlType = typeof(SettingsSlider<double, PercentSlider>))]
+        public BindableNumber<double> MinimumAccuracy { get; } = new BindableDouble
+        {
+            MinValue = 0.01,
```

Comment:
This seems silly. Maybe 60%? 80%? Definitely not 1%.

Reply:
60% it is then.